demo/JENKINS_HOME/credentials.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<com.cloudbees.plugins.credentials.SystemCredentialsProvider plugin="credentials@1.25">
```

Comment:
FYI these `plugin` attributes are not used for anything interesting and may be omitted from demo images if you want to be tidy.